packages/strapi-admin/controllers/Auth.js

```diff
@@ -188,12 +187,10 @@ module.exports = {
       });
     } catch (err) {
       strapi.log.error(err);
-      const adminError = _.includes(err.message, 'username')
-        ? {
-            id: 'Auth.form.error.username.taken',
-            message: 'Username already taken',
-          }
-        : { id: 'Auth.form.error.email.taken', message: 'Email already taken' };
+      const adminError = {
+        id: 'Auth.form.register.error',
```

Comment:
This would also return database errors which isn't a good practice (even if it is done somewhere else in the code).

Reply:
Thanks a bunch for your feedback, @alexandrebodin .
I made a slightly change on the code in order to return only the stack's first line.
In my case, in order to reproduce the problem I've created a number field on User table which it is called "age".
The new error message it is gonna look like this:

    "message": [
        {
            "messages": [
                {
                    "id": "Auth.form.register.error",
                    "message": "Error: SQLITE_CONSTRAINT: NOT NULL constraint failed: users-permissions_user.age"
                }
            ]
        }
    ]

Do you think it is a good enough solution?
I hope that might make sense.